### ISSUE TITLE
XCode Run Script step broken after commit aeb465339afed5b6f5a9ae2d06ffc7853467609f

### ISSUE THREAD

**Issue by rafaelmaroxa** [SwiftKit/Cuckoo#61]
I needed the fix for #55, so I updated to 0.7.0, which then broke the XCode build step (see logs below). I then went back and reinstalled my pods pointing Cuckoo to every commit before the fix for #55. It turns out that commit aeb465339afed5b6f5a9ae2d06ffc7853467609f broke the script that I had, which I copied out of the Readme.md.

```
Generated Mocks File = ./XYZTests/Mocks/GeneratedMocks.swift
Mocks Input Directory = ./XYZ/Code
/Users/.../XYZ/Pods/Cuckoo/build_generator: line 4: cd: Generator: No such file or directory
Build settings from command line:
    CONFIGURATION_BUILD_DIR = /Users/.../XYZ/Build

xcodebuild: error: 'CuckooGenerator.xcodeproj' does not exist.
mv: rename Build/cuckoo_generator.app to cuckoo_generator.app: No such file or directory
/Users/.../XYZ/smartcom-swift/Pods/Cuckoo/run: line 11: /Users/.../XYZ/Pods/Cuckoo/Generator/cuckoo_generator.app/Contents/MacOS/cuckoo_generator: No such file or directory
```

**Comment by TadeasKriz** [comment]
Did `pod install` work correctly? Could you go to `Pods/Cuckoo` and see if there is `CuckooGenerator` directory?

**Comment by rafaelmaroxa** [comment]
Thanks for the quick response!

`pod install` did work correctly. I am using `pod 'Cuckoo', git: 'https://github.com/SwiftKit/Cuckoo'` to get 0.7.0, because it seems like the podspec hasn't been updated in the Cocoapods repo yet.

When I get 0.7.0 and navigate to the `Pods/Cuckoo` directory, this is all I see:
![image](https://cloud.githubusercontent.com/assets/650942/18387922/f822b422-7664-11e6-9542-1c652ed1c65f.png)

**Comment by TadeasKriz** [comment]
1) If you have CocoaPods >=1.0, run `pod repo update`, then `pod 'Cuckoo', '0.7.0'` should work.

2) I'll look at it :)

**Comment by TadeasKriz** [comment]
@rafaelmaroxa Could you please get me a screenshot of the `Generator` directory? Thanks.

**Comment by rafaelmaroxa** [comment]
Will update central repo and point to 0.7.0. Thanks!

**Comment by rafaelmaroxa** [comment]
Here it is:
![image](https://cloud.githubusercontent.com/assets/650942/18388214/647b8b2a-7666-11e6-908c-5e9a18026709.png)

**Comment by TadeasKriz** [comment]
I think I see the problem, let me make an edit and get back to you with a commit to test :)

**Comment by TadeasKriz** [comment]
@rafaelmaroxa Please try commit: `aa62389e6e5addeadb78a9ec729a9005e4f3c20c` and let me know.

**Comment by rafaelmaroxa** [comment]
Sweet!

**Comment by rafaelmaroxa** [comment]
It worked great. Thanks ;)

**Comment by TadeasKriz** [comment]
@FilipDolnik Could you test this as well and make sure it works as expected? Thanks.

**Comment by TadeasKriz** [comment]
@rafaelmaroxa No problem :) I'll be released in `0.7.1`.
